docker-compose.yml

```
@@ -36,9 +36,11 @@ services:
       context: .
     ports:
       - 8000:8000
-    # PROMETHEUS_MULTIPROC_DIR is required for multiple workers, see:
-    # https://prometheus.github.io/client_python/multiprocess/
     environment:
+      # list here usernames which should be assigned SUPERUSER role on application start
+      SYNCMASTER__ENTRYPOINT__SUPERUSERS: syncmaster
```

```suggestion
      SYNCMASTER__ENTRYPOINT__SUPERUSERS: admin
```

IMHO this makes more sense